tests/lib/nsxv_test.py

```diff
@@ -337,23 +385,45 @@ def testStrForinet6(self):
 
   def testTranslatePolicy(self):
     """Test for Nsxv.test_TranslatePolicy."""
-    # exp_info default is 2
     self.naming.GetNetAddr('NTP_SERVERS').AndReturn([nacaddr.IP('10.0.0.1'),
                                                      nacaddr.IP('10.0.0.2')])
     self.naming.GetNetAddr('INTERNAL').AndReturn([nacaddr.IP('10.0.0.0/8'),
                                                   nacaddr.IP('172.16.0.0/12'),
                                                   nacaddr.IP('192.168.0.0/16')])
     self.naming.GetServiceByProto.return_value = ['123']
 
-    exp_info = 2
     pol = policy.ParsePolicy(INET_FILTER, self.naming, False)
-    translate_pol = nsxv.Nsxv(pol, exp_info)
+    translate_pol = nsxv.Nsxv(pol, EXP_INFO)
+    nsxv_policies = translate_pol.nsxv_policies
+    for (header, filter_name, filter_list, section_id, terms) in nsxv_policies:
+      self.assertEqual(filter_name, 'inet')
+      self.assertEqual(filter_list, ['inet'])
+      self.assertEqual(len(terms), 1)
+
+    self.naming.GetServiceByProto.assert_has_calls(
+        [mock.call('NTP', 'udp')] * 2)
+
+  def testTranslatePolicyWithEstablished(self):
+    """Test for Nsxv.test_TranslatePolicy."""
+    # exp_info default is 2
+    self.naming.GetNetAddr('NTP_SERVERS').AndReturn([nacaddr.IP('10.0.0.1'),
+                                                     nacaddr.IP('10.0.0.2')])
+    self.naming.GetNetAddr('INTERNAL').AndReturn([nacaddr.IP('10.0.0.0/8'),
+                                                  nacaddr.IP('172.16.0.0/12'),
+                                                  nacaddr.IP('192.168.0.0/16')])
+    self.naming.GetServiceByProto.return_value = ['123']
+
+    pol = policy.ParsePolicy(INET_FILTER_WITH_ESTABLISHED, self.naming, False)
+    translate_pol = nsxv.Nsxv(pol, EXP_INFO)
     nsxv_policies = translate_pol.nsxv_policies
-    for (_, filter_name, filter_list, terms) in nsxv_policies:
+    for (header, filter_name, filter_list, section_id, terms) in nsxv_policies:
```

Comment:
Same here, please adopt our style guide and use underscores to mark variables that are unused.